samples/ballerina-by-example/examples/websocket-basic-endpoint/websocket-basic-endpoint.bal

```diff
@@ -0,0 +1,29 @@
+import ballerina.lang.system;
+import ballerina.lang.messages;
+import ballerina.net.http;
+import ballerina.net.ws;
+import ballerina.doc;
+
+@doc:Description {value:"WebSocket endpoint is defined as a composition of  BasePath + WebSocketUpgradePath"}
+@http:BasePath {value:"/endpoint"}
+@ws:WebSocketUpgradePath {value:"/ws"}
+service websocketServerEndpoint {
+
+    @doc:Description {value:"@OnOpen annotation is triggered when a new connection is initiated."}
+    @ws:OnOpen {}
+    resource onOpen(message m) {
+        system:println("New client connected to the server.");
```

Comment:
This line goes beyond the source code box when generated the HTML page using build tool.